Index: mysite/mysite/settings.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mysite/mysite/settings.py b/mysite/mysite/settings.py
--- a/mysite/mysite/settings.py	
+++ b/mysite/mysite/settings.py	(date 1689414744965)
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [BASE_DIR],
+        "DIRS": [BASE_DIR / "mysite/templates"],
         "APP_DIRS": True,
         "OPTIONS": {
             "context_processors": [
